server/jetty/build.gradle

```diff
@@ -11,7 +11,10 @@ dependencies {
         because 'downstream dagger compile'
     }
 
-    implementation project(":util-thread")
+    implementation project(':util-thread')
+    compileOnlyApi(project(':util-thread')) {
+        because 'downstream dagger compile'
+    }
```

Comment:
Thank you - I ran into this the other day. Want me to submit this in its own PR?

Reply:
I'm not too worried; if this merges in soon, should be ok here.